app/src/androidTest/java/com/sdp13epfl2021/projmag/database/UserProfileDatabaseTest.kt

```diff
@@ -113,9 +113,9 @@ class UserProfileDatabaseTest {
     @Test
     fun checkThatGettingProfileWorks() {
         val udb = UserProfileDatabase(mockFirebaseFirestore, mockFirebaseAuth)
-        udb.getProfile {
+        udb.getProfile({
             assert(it != null)
```

Comment:
Using an assertEquals would be better than just null. (need to add "data" to the ImmutableProfile class)
`assertEquals(exampleUserProfile, it)`

Reply:
Indeed thats much better